api/agent/lb_agent.go

```diff
@@ -364,7 +364,7 @@ func recordCallLatency(ctx context.Context, call *call, status string) {
 	// some sanity check before. If sanity checks flags something, then
 	// this is likely that runners are sending malicious/suspicious data.
 	if execLatency != nil {
-		if *execLatency >= callLatency {
+		if *execLatency-callLatency >= 2*time.Millisecond {
```

Comment:
a comment here about why 2ms might be nice, out of context this comparison doesn't make any sense. flooring to the nearest millisecond would perhaps read more clearly and not require a comment. go has decent facilities for this (https://golang.org/pkg/time/#Duration.Truncate) eg:

```go
if (*execLatency).Truncate(time.Millisecond) >= callLatency.Truncate(time.Millisecond)
```

Reply:
I think Truncate() strips monotonic clock, so I don't know if we want that here.

Reply:
it's a simple floor: https://golang.org/src/time/time.go?s=25865:25912#L801 -- we should do `>` and not `>=` though, good catch